### ISSUE TITLE
redmine_ckeditor version 1.0.20 is not compatible with Ruby 1.9.2

### ISSUE THREAD

**Issue by enthurohini** [a-ono/redmine_ckeditor#238]
Here is my Redmine setup Information:

> Environment:
>   Redmine version            2.4.2.stable
>   Ruby version                  1.9.3-p484 (2013-11-22) [x86_64-linux]
>   Rails version                   3.2.16
>   Environment                   production
>   Database adapter           MySQL
> SCM:
>   Subversion                     1.8.8
>   Git                                  1.9.1
>   Filesystem

I clone the  redmine_ckeditor repository and checkedout to the version 1.0.20. But when I do
`rake redmine:plugins:migrate RAILS_ENV=production`

It gives error:

> Could not find gem 'ruby (>= 2.2.2)', which is required by gem 'paperclip (~> 4.2.1)', in any of the sources.

**Comment by a-ono** [comment]
Are you sure you've checked out version 1.0.20?
Please confirm your Gemfile.
https://github.com/a-ono/redmine_ckeditor/blob/1.0.20/Gemfile#L4

**Comment by enthurohini** [comment]
Yes, my Gemfile is same as [https://github.com/a-ono/redmine_ckeditor/blob/1.0.20/Gemfile#L4](https://github.com/a-ono/redmine_ckeditor/blob/1.0.20/Gemfile#L4). And now error is:

> Could not find gem 'ruby (>= 2.2.2)', which is required by gem 'activemodel (>= 3.0.0)', in any of the sources.

**Comment by a-ono** [comment]
Your Redmine does not seem to be version 2.4.2.
Please check [Redmine's Gemfile](https://github.com/redmine/redmine/blob/2.4.2/Gemfile).
And you should execute `bundle install` in the **Redmine's root** directory.

**Comment by enthurohini** [comment]
I have attached my Redmine setup information already, it is **2.4.2.stable**. Ans yes, I am executing `bundle install` in the **Redmine's root directory**. Still getting above mentioned error. Please help!

**Comment by a-ono** [comment]
Please try `bundle update`.
